internal/controllers/core/cmd/controller_test.go

```diff
@@ -941,14 +1057,14 @@ func (f *fixture) assertCmdMatches(name string, matcher func(cmd *Cmd) bool) *Cm
 		return matcher(cmd)
 	}, timeout, interval)
 
-	return f.assertCmdMatchesInAPI(name, matcher)
+	return f.requireCmdMatchesInAPI(name, matcher)
 }
 
-func (f *fixture) assertCmdMatchesInAPI(name string, matcher func(cmd *Cmd) bool) *Cmd {
+func (f *fixture) requireCmdMatchesInAPI(name string, matcher func(cmd *Cmd) bool) *Cmd {
```

Comment:
I made this change because I was getting spurious / misleading error messages from subsequent code when the assertion failed, and terminating seems better. If someone has a preference for `assert` mechanics in this method, 
I can do it another way.